extensions/web/pkg/config/http.go

```diff
@@ -2,8 +2,8 @@ package config
 
 // HTTP defines the available http configuration.
 type HTTP struct {
-	Addr      string `yaml:"addr" env:"WEB_HTTP_ADDR"`
+	Addr      string `yaml:"addr" env:"WEB_HTTP_ADDR" desc:"The bind address of the HTTP service."`
 	Namespace string `yaml:"-"`
-	Root      string `yaml:"root" env:"WEB_HTTP_ROOT"`
-	CacheTTL  int    `yaml:"cache_ttl" env:"WEB_CACHE_TTL"`
+	Root      string `yaml:"root" env:"WEB_HTTP_ROOT" desc:"The root path of the HTTP service."`
+	CacheTTL  int    `yaml:"cache_ttl" env:"WEB_CACHE_TTL" desc:"Cache policy for ownCloud Web assets."`
```

Comment:
The `WEB_HTTP_ROOT` default is `/`. It could be set to any sub-path, eg `/foo/baz/123`, but I hope no one ever will use this option.

The `WEB_CACHE_TTL` default is 604800, which equals a cache ttl of 7 days